test/compile/mark-point.test.ts

```diff
@@ -114,7 +114,15 @@ describe('Mark: Square', function() {
   });
 
   it('should be filled by default', function() {
-    // TODO
+    const model = parseModel({
```

Comment:
Shouldn't the cost model and props be defined outside it()? Looking at the rest of the tests I think it should be defined inside the describe()